### DISCUSSION TITLE
Calling channelMap.remove from the Postprocessor

### DISCUSSION THREAD

**Discussion by JonEBee** [nextgenhealthcare/connect#4620] [Q&A] [answered]
Mirth 3.8.1 stopped execution of my script with a "`Wrapped java.lang.UnsupportedOperationException`" for the following line:

`981: channelMap.remove('apiResponse');`

Kind of odd to me, since the `get` and `put` methods work in there.

Just wanted to confirm that this is not a bug and the `remove` method is not allowed in the Postprocessor.

Thanks.

**Comment by JonEBee** [comment, marked as answer]
I figured it out...never mind.

**Reply by tonygermano** [reply]
If you aren't going to share what the problem was, can you at least mark this as the accepted answer so it doesn't linger as an unanswered question?

**Reply by JonEBee** [reply]
I was the only problem.  ChannelMap methods simply cannot be executed in the Postprocessor environment and I figured that out after creating this conversation.
